package.json

```diff
@@ -19,6 +20,7 @@
     "framer-motion": "^11.3.2",
     "lucide-react": "^0.407.0",
     "next": "14.2.4",
+    "posthog-node": "^4.0.1",
```

Comment:
Analytics events are usually sent from the client side. I think we should use `posthog-js`